src/main/java/Problem2/LCA.java

```diff
@@ -4,8 +4,23 @@
 
 public class LCA {
     public static TreeNode<Integer> lowestCommonAncestor(TreeNode<Integer> root, TreeNode<Integer> p, TreeNode<Integer> q) {
-        // homework
-        // hint: pay attention to the assumptions of this problem.
-        return null;    // place holder
+        if (root == null) {
+            return null;
+        }
+        if (root == p || root == q) {
```

Comment:
you want to check if their values are the same, not the entire TreeNode

Reply:
It is checking reference to node not entire tree.

Reply:
the input parameter is the left and right node and not the value